### PR TITLE
Feat/usage

### PR DESCRIPTION
**Pull request by zuoxiaodong0815**

### CODE REVIEW

File: apps/cloud/src/api/blobs.rs [hunk 1]
```diff
@@ -143,6 +155,31 @@ impl Context {
...
+        let Ok(workspace_id_list) = self.db.get_user_owner_workspaces(user_id).await else {
+            return Err(ErrorStatus::InternalServerError);
+        };
+        let mut total_size = 0;
+        for workspace_id in workspace_id_list {
+            let Ok(size) = self
+                    .storage
+                    .blobs()
+                    .get_blobs_size(workspace_id.to_string())
+                    .await else {
```
Comment (Brooooooklyn) [inline apps/cloud/src/api/blobs.rs:171]: I'd like to pass more error messages here, could that be done in the current implementation?  `let size = self.storage.blobs().get_blobs_size(workspace_id.to_string()).await.map_err(|err| ErrorStatus::InternalServerError(err))?;`

**Review by darkskygit — APPROVED** [review]

File: apps/cloud/src/api/blobs.rs [hunk 2]
```diff
@@ -143,6 +155,31 @@ impl Context {
...
+                        return Err(ErrorStatus::InternalServerError);
+                    };
+            total_size += size as u64;
+        }
+        let blob_usage = BlobUsage {
+            usage: total_size,
+            max_usage: 10 * 1024 * 1024 * 1024,
```
[the remark this answers is not in the document]
Comment (zuoxiaodong0815) [reply apps/cloud/src/api/blobs.rs]: Yes,I have modified this.

File: apps/cloud/src/api/blobs.rs (hunk 1 above)
Comment (zuoxiaodong0815) [reply apps/cloud/src/api/blobs.rs]: Ok, it has been modified.

**Review by Himself65 — COMMENTED** [review]
I didn't see the API in http://localhost:3000/api/docs/#/

Anything wrong?

**Pull request merged**